utils/sites.js

```diff
@@ -596,8 +596,93 @@ const participantDataCorrection = async (req, res) => {
     }
 };
 
+const getBigQueryData = async (req, res) => {
+    const { validateTableAccess, validateFilters, validateFields, getBigQueryData } = require('./bigquery');
+
+    logIPAddress(req);
+    setHeaders(res);
+    
+    if(req.method === 'OPTIONS') return res.status(200).json({code: 200});
+
+    if(req.method !== 'GET') {
+        return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+    }
+    // const { APIAuthorization } = require('./shared');
+    // const authorized = await APIAuthorization(req);
+    // if(authorized instanceof Error){
+    //     return res.status(500).json(getResponseJSON(authorized.message, 500));
+    // }
+
+    // if(!authorized){
+    //     return res.status(401).json(getResponseJSON('Authorization failed!', 401));
+    // }
```

Comment:
Does this need to be removed?

Reply:
It needed to be reinstated I had it commented out for local testing.